From patches Mon Sep 17 00:00:00 2001
From: peter.crosthwaite@xilinx.com
To: qemu-devel@nongnu.org
Cc: edgar.iglesias@gmail.com, peter.maydell@linaro.org
Date: Thu, 18 Apr 2013 14:46:53 +1000
Message-id: f95153bc6a14a2f745639fd77c0095fd010e0465.1366260005.git.peter.crosthwaite@xilinx.com
Subject: [PATCH 2/3] dtc: add submodule

From: Peter Crosthwaite <peter.crosthwaite@xilinx.com>

Add dtc submodule as a fallback for old distros.
Picking version 1.3.0. as this is the most recently tagged stable version.

Signed-off-by: Peter Crosthwaite <peter.crosthwaite@xilinx.com>
Reviewed-by: Peter Maydell <peter.maydell@linaro.org>
---

 .gitmodules |    3 +++
 dtc         |    1 +
 2 files changed, 4 insertions(+), 0 deletions(-)
 create mode 160000 dtc

diff --git a/.gitmodules b/.gitmodules
index cfa2af9..b426ea6 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -22,3 +22,6 @@
 [submodule "pixman"]
 	path = pixman
 	url = git://anongit.freedesktop.org/pixman
+[submodule "dtc"]
+	path = dtc
+	url = git://git.qemu.org/dtc.git
diff --git a/dtc b/dtc
new file mode 160000
index 0000000..bc895d6
--- /dev/null
+++ b/dtc
@@ -0,0 +1 @@
+Subproject commit bc895d6d09695d05ceb8b52486ffe861d6cfbdde
-- 
1.7.0.4


